apps/laboratory/src/pages/index.tsx

```diff
@@ -18,6 +18,8 @@ import {
 } from 'wagmi/chains'
 import { ConnectButton } from '../components/ConnectButton'
 import { NetworksButton } from '../components/NetworksButton'
+import useThemeStore from '../utils/StoreUtil'
+import { setThemeVariables } from '@web3modal/ui'
```

Comment:
ui, core and scafold packages should never be exposed to end users, they should opperate on theme only from `@web3modal/wagmi` packages.

I believe this is here to set up initial theme? If so, this should be done inside `createModal` function or `new Web3Modal()` constructor. Subsequent updates are done with hook or method on web3modal, like you use setThemeMode here.